blocks/user-info/user-info.js

```diff
@@ -9,9 +9,8 @@ export default async function decorate(block) {
   const DYNAMIC_365_DOMAIN = getEnvironmentConfig('dev').profile.dynamic365domain;
 
   // TODO move to template logic
-  if (userId == null || userId === undefined || accessToken == null || accessToken === undefined) {
-    window.location.assign(`${window.location.origin}`);
-  } else {
+  // eslint-disable-next-line max-len
+  if (userId !== null || userId !== undefined || accessToken !== null || accessToken !== undefined) {
```

Comment:
It's unusual to compare with `null` and  `undefined` directly. 

```suggestion
  if (!userId || !accessToken) {
```

Reply:
@shehjadkhan-ansys FYI, to avoid comparing with `null` and `undefined` directly

Reply:
Sure @synox, thank you explaining me

Reply:
fixed